<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QWidget" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>611</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>10</horstretch>
     <verstretch>10</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tabLive">
    <attribute name="title">
     <string>Live mode</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>791</width>
       <height>581</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="boxLive"/>
    </widget>
   </widget>
   <widget class="QWidget" name="tabAnalyse">
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <attribute name="title">
     <string>Analyse mode</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>791</width>
       <height>561</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="boxAnalyse">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="boxButtonsAnalyse">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="checkT5">
            <property name="text">
             <string>Thermocouple 5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkT4">
            <property name="text">
             <string>Thermocouple 4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkT3">
            <property name="text">
             <string>Thermocouple 3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkT2">
            <property name="text">
             <string>Thermocouple 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkT1">
            <property name="text">
             <string>Thermocouple 1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="btnUpdate">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="boxPlotAnalyse"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Concrete mode</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>721</width>
       <height>571</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Pictures/radiateur.jpg</pixmap>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioT1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>T1</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>44</width>
       <height>30</height>
      </size>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdTemp">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioT5">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>450</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>T5</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioT3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>T3</string>
     </property>
     <property name="styleSheet">
      <string notr="true">
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioT2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>150</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>T2</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioT4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>320</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>T4</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
